compliance/cis_k8s/rules/cis_1_1_13/data.yaml

```diff
@@ -0,0 +1,45 @@
+metadata:
+  id: 1aa16467-18ce-5ba6-8e6a-1f6b69acbfa0
+  name: Ensure that the admin.conf file permissions are set to 644 or more
+    restrictive (Automated)
+  profile_applicability: |
+    * Level 1 - Master Node
+  description: >
+    Ensure that the `admin.conf` file has permissions of `644` or more
```

Comment:
we haven't define `description` and other fields as markdown (only `remediation`, `audit` and `references`), I guess it would be fine. I'll open later a followup ticket to present that properly on kibana.